### PR TITLE
Added guide to edit variables

### PR DESCRIPTION
**Pull request by AntonWetzel**

### CODE REVIEW

File: tutorials/misc/running_code_in_the_editor.rst [hunk 1]
```diff
@@ -88,6 +88,27 @@ Now let's choose which code runs when. Modify your ``_process()`` function to lo
...
+Add and export a variable speed to the script. The function set_speed after "setget" is executed with your input to change the variable.
+Modify  ``_process()`` to include the rotation speed.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    tool
+    extends Sprite
+
+    export var speed: float = 1   setget set_speed
```
Comment (Feniks-Gaming) [inline tutorials/misc/running_code_in_the_editor.rst]: Don't use static typing I  your examples.

File: tutorials/misc/running_code_in_the_editor.rst [hunk 2]
```diff
@@ -88,6 +88,27 @@ Now let's choose which code runs when. Modify your ``_process()`` function to lo
...
+
+    #update the speed and reset the rotation
```
Comment (Feniks-Gaming) [inline tutorials/misc/running_code_in_the_editor.rst]: Style Guide suggests that you leave a space between # and update `# Update`

File: tutorials/misc/running_code_in_the_editor.rst [hunk 3]
```diff
@@ -88,6 +88,27 @@ Now let's choose which code runs when. Modify your ``_process()`` function to lo
...
+    func set_speed(new_speed: float):
+    	speed = new_speed
+    	rotation_degrees = 0
+
```
Comment (Feniks-Gaming) [inline tutorials/misc/running_code_in_the_editor.rst]: Style Guide requires 2 blank lines between functions you have only one

File: tutorials/misc/running_code_in_the_editor.rst [hunk 4]
```diff
@@ -88,6 +88,27 @@ Now let's choose which code runs when. Modify your ``_process()`` function to lo
...
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    tool
+    extends Sprite
+
+    export var speed: float = 1   setget set_speed
+
+    #update the speed and reset the rotation
+    func set_speed(new_speed: float):
```
Comment (Feniks-Gaming) [inline tutorials/misc/running_code_in_the_editor.rst]: Same as above don't use static typing.

File: tutorials/misc/running_code_in_the_editor.rst [hunk 5]
```diff
@@ -88,6 +88,28 @@ Now let's choose which code runs when. Modify your ``_process()`` function to lo
...
+-----------------
+Add and export a variable speed to the script. The function set_speed after "setget" is executed with your input to change the variable.
+Modify  ``_process()`` to include the rotation speed.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    tool
+    extends Sprite
+
```
Comment (YeldhamDev) [inline tutorials/misc/running_code_in_the_editor.rst]: Add an extra line here.

File: tutorials/misc/running_code_in_the_editor.rst [hunk 6]
```diff
@@ -88,6 +88,28 @@ Now let's choose which code runs when. Modify your ``_process()`` function to lo
...
+    export var speed = 1   setget set_speed
+
```
Comment (YeldhamDev) [inline tutorials/misc/running_code_in_the_editor.rst]: Extra line here as well.

File: tutorials/misc/running_code_in_the_editor.rst [hunk 7]
```diff
@@ -88,6 +88,28 @@ Now let's choose which code runs when. Modify your ``_process()`` function to lo
...
+Add and export a variable speed to the script. The function set_speed after "setget" is executed with your input to change the variable.
+Modify  ``_process()`` to include the rotation speed.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    tool
+    extends Sprite
+
+    export var speed = 1   setget set_speed
```
Comment (YeldhamDev) [inline tutorials/misc/running_code_in_the_editor.rst]: Just one space before `setget`, not three.

File: tutorials/misc/running_code_in_the_editor.rst [hunk 8]
```diff
@@ -88,6 +88,28 @@ Now let's choose which code runs when. Modify your ``_process()`` function to lo
...
+
+    # Update speed and reset the rotation
```
Comment (YeldhamDev) [inline tutorials/misc/running_code_in_the_editor.rst]: Add a period at the end of the comment.

File: tutorials/misc/running_code_in_the_editor.rst (hunk 5 above)
Comment (mhilbrunner) [inline tutorials/misc/running_code_in_the_editor.rst:101]:
```suggestion


```

File: tutorials/misc/running_code_in_the_editor.rst (hunk 7 above)
Comment (mhilbrunner) [inline tutorials/misc/running_code_in_the_editor.rst:102]:
```suggestion
    export var speed = 1 setget set_speed
```

File: tutorials/misc/running_code_in_the_editor.rst (hunk 6 above)
Comment (mhilbrunner) [inline tutorials/misc/running_code_in_the_editor.rst:103]:
```suggestion


```

File: tutorials/misc/running_code_in_the_editor.rst (hunk 8 above)
Comment (mhilbrunner) [inline tutorials/misc/running_code_in_the_editor.rst:104]:
```suggestion
    # Update speed and reset the rotation.
```

**Comment by mhilbrunner** [comment]
Applied the suggestions by @YeldhamDev and merged. Thanks for this, and sorry it took a little to get merged!

Congrats on your first docs contribution :)

**Pull request merged**